Prose/ProseLib/Sources/ProseCore/ProseClient.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import ProseCoreClientFFI
+
+enum ProseClientError: Error {
+    case missingCredentials
+}
```

Comment:
Can you please localize it as https://github.com/prose-im/prose-app-macos/pull/67/files#diff-42b4865f1cfc666f7bc399a955aa405e68bbb26f6c372ac38dddb77428bfed82 was? (The localizable string is still here)

Reply:
Oh but you probably don't want to add AppLocalization as a dependency to `ProseClient`, so `ProseClient` should have its own `localizable.strings`.

Reply:
The library is in flux and I want to stay nimble by not introducing documentation and localization which needs to be changed and refactored as the library changes.